SU2_CFD/src/variables/CNSVariable.cpp

```diff
@@ -114,6 +114,8 @@ void CNSVariable::SetRoe_Dissipation_FD(unsigned long iPoint, su2double val_wall
   AD::SetPreaccIn(Primitive(iPoint, indices.EddyViscosity()));
   /*--- Laminar viscosity --- */
   AD::SetPreaccIn(Primitive(iPoint, indices.LaminarViscosity()));
+  /*--- Density ---*/
+  AD::SetPreaccIn(Solution(iPoint, 0));
```

Comment:
```suggestion
  AD::SetPreaccIn(Solution(iPoint, indices.Density()));
```

Reply:
indices.Density() is for the position in `Primitive`, I think..

Reply:
what a pain this code :( then please mention in the comment that this is for `GetDensity` below, because if we change that to get density from somewhere else it breaks.